docs/adguard-for-android/solving-problems/compatibility-issues.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Known compatibility issues with Android apps
+sidebar_position: 14
+---
+
+### VPN apps
+
+Other VPN apps are incompatible with AdGuard running in the Local VPN filtering mode since you cannot have two active VPNs at the same time. To solve this problem we suggest you:
+
+* use AdGuard VPN — it has Integrated mode that allows two apps to operate seamlessly
+* use AdGuard in Local HTTP Proxy mode. *Please note that 'manual' HTTP Proxy mode is not supported on devices since Android 10*
+* if you use a third-party VPN app, figure out whether it supports SOCKS4/SOCKS5 mode. If it does, you can configure your VPN app to work as outbound proxy and set up local outbound proxy with params from the third-party app. For instance, AdGuard can be automatically configured with 'Orbot: Proxy with Tor'
+
+### Private DNS
+
+The Private DNS feature was introduced in Android Pie. Before version Q, Private DNS didn't break our logic and the DNS forwarding through AdGuard worked normally. But starting from version Q, the presence of Private DNS forces apps to redirect traffic through the system resolver instead of AdGuard. See Android [devs blog](https://android-developers.googleblog.com/2018/04/dns-over-tls-support-in-android-p.html) for more details.
```

Comment:
It is still 'our' logic instead. Maybe we can use 'AdGuard DNS filtering logic' or something?